### PR TITLE
mcomix profile creation

### PR DESCRIPTION
**Pull request by pirate486743186**
new profile

copied a lot from zathura

### CODE REVIEW

File: etc/profile-m-z/mcomix.profile [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/mcomix
+mkdir ${HOME}/.local/share/mcomix
+whitelist /usr/share/mcomix
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile:34]: `include whitelist-runuser-common.inc`?

File: etc/profile-m-z/mcomix.profile [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+noblacklist ${HOME}/.config/mcomix
+noblacklist ${HOME}/.local/share/mcomix
+noblacklist ${DOCUMENTS}
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+# Allow python (blacklisted by disable-interpreters.inc)
+#1.2
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile:17]: 1.2?

File: etc/profile-m-z/mcomix.profile [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile:28]: If it's only a viewer, we can `include disable-write-mnt.inc`.

File: etc/profile-m-z/mcomix.profile [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+nogroups
+noinput
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix
+seccomp
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile:50]: `seccomp.block-secondary`?

File: etc/profile-m-z/mcomix.profile (hunk 2 above)
Comment (pirate486743186) [reply etc/profile-m-z/mcomix.profile:17]: version 1.2 is the last version with python 2 and gtk2

File: etc/profile-m-z/mcomix.profile [hunk 5]
```diff
@@ -26,12 +26,14 @@ include disable-passwdmgr.inc
 include disable-programs.inc
 include disable-shell.inc
 include disable-xdg.inc
+include disable-write-mnt.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile]:
```suggestion
include disable-write-mnt.inc
include disable-xdg.inc
```

File: etc/profile-m-z/mcomix.profile [hunk 6]
```diff
@@ -14,7 +14,7 @@ noblacklist ${DOCUMENTS}
 include allow-bin-sh.inc
 
 # Allow python (blacklisted by disable-interpreters.inc)
-#1.2
+#mcomix =< 1.2
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile:17]: and what's with mcomix >= 1.2?

File: etc/profile-m-z/mcomix.profile [hunk 7]
```diff
@@ -47,11 +49,11 @@ notv
 nou2f
 novideo
 protocol unix
-seccomp
+seccomp.block-secondary
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile:52]:
```suggestion
seccomp
seccomp.block-secondary
```

File: etc/profile-m-z/mcomix.profile [hunk 8]
```diff
@@ -47,11 +49,11 @@ notv
...
 shell none
 tracelog
 
-#1.2 python2
+#mcomix =< 1.2 python2
```
Comment (rusty-snake) [inline etc/profile-m-z/mcomix.profile:56]:
```suggestion
# mcomix <= 1.2 uses python2
```

if we want such notes.

File: etc/profile-m-z/mcomix.profile (hunk 4 above)
Comment (rusty-snake) [reply etc/profile-m-z/mcomix.profile]: `seccomp.block-secondary` does not imply `seccomp`

File: etc/profile-m-z/mcomix.profile (hunk 6 above)
Comment (pirate486743186) [reply etc/profile-m-z/mcomix.profile:17]: upgraded to python 3  / gtk 3

**Comment by rusty-snake** [comment]
merged, thanks.

**Pull request merged**